NetCoreDemos/Views/Splitter/Overview.cshtml

```diff
@@ -2,56 +2,61 @@
     <div class="demo-container">
         @(Html.DevExtreme().Splitter()
             .ID("splitter")
-            .Items(items => {
+            .Items(items =>
+            {
                 items.Add()
-                    .Resizable(true)
-                    .Size("140px")
-                    .MinSize("70px")
-                    .Template(new JS("function(data){return paneContentTemplate(data, 'Left Pane')}"));
+                .Resizable(true)
+                .Size("140px")
+                .MinSize("70px")
+                .Template(new JS("function(data, itemIndex, itemElement){return paneContentTemplate(data, 'Left Pane', itemElement)}"));
 
-                items.Add().Splitter(s => {
+                items.Add().Splitter(s =>
+                {
                     s.Orientation(Orientation.Vertical);
-                    s.Items(verticalItems => {
+                    s.Items(verticalItems =>
+                    {
                         verticalItems.Add()
-                            .Resizable(true)
-                            .Collapsible(true)
-                            .MaxSize("75%")
-                            .Template(new JS("function(data){return paneContentTemplate(data, 'Central Pane')}"));
+                        .Resizable(true)
+                        .Collapsible(true)
+                        .MaxSize("75%")
+                        .Template(new JS("function(data, itemIndex, itemElement){return paneContentTemplate(data, 'Central Pane', itemElement)}"));
 
                         verticalItems.Add()
-                            .Collapsible(true)
-                            .Splitter(horizontalSplitter => {
-                                horizontalSplitter.Orientation(Orientation.Horizontal);
-                                horizontalSplitter.Items(horizontalItems => {
-                                    horizontalItems.Add()
-                                        .Resizable(true)
-                                        .Collapsible(true)
-                                        .Size("30%")
-                                        .MinSize("5%")
-                                        .Template(new JS("function(data){return paneContentTemplate(data, 'Nested Left Pane')}"));
+                        .Collapsible(true)
+                        .Splitter(horizontalSplitter =>
+                        {
+                            horizontalSplitter.Orientation(Orientation.Horizontal);
+                            horizontalSplitter.Items(horizontalItems =>
+                            {
+                                horizontalItems.Add()
+                    .Resizable(true)
```

Comment:
I see a lot if incorrect indents in this file. Fix it please

Reply:
[28ba54b](https://github.com/DevExpress/devextreme-demos/pull/3206/commits/28ba54bc841c98cb4dae42826a2ad9274c6c4ad8)